fed/mosaic-sumo/src/main/java/org/eclipse/mosaic/fed/sumo/ambassador/SumoAmbassador.java

```diff
@@ -122,15 +118,19 @@ public SumoAmbassador(AmbassadorParameter ambassadorParameter) {
     /**
      * Find the name of the route file and return it.
      *
-     * @param sumoConfigurationFile The SUMO configuration file.
-     * @return The route-files filename.
+     * @return The route-file name.
      */
-    private String parseRouteFilename(File sumoConfigurationFile) {
-
+    private String setRouteFileNames() {
         try {
             XMLConfiguration sumoConfiguration = XmlUtils.readXmlFromFile(sumoConfigurationFile);
-            String routeFile = XmlUtils.getValueFromXpath(sumoConfiguration, "/input/route-files/@value", null);
-            return Validate.notNull(routeFile);
+            String routeFileName = XmlUtils.getValueFromXpath(sumoConfiguration, "/input/route-files/@value", null);
+            Validate.notNull(routeFileName);
+            String[] routeFiles = routeFileName.split("[\\s,]+"); // split by comma (+ white space)
+            if (routeFiles.length > 1) {
+                log.debug("It seems like there was more than one route-file defined.");
```

Comment:
What is the consequence?

Reply:
Actually there isn't really one. I'm going to remove this.